test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -57,6 +57,19 @@
     ('aten::Int', datetime.date(2020, 1, 30)),
 ]
 
+jit_test_functions = [
+    '_TorchScriptTesting_StackString::pop',
```

Comment:
Nit: Also, just use `_TorchScriptTesting_` should be enough, and you can directly add this to the whitelist, and add a comment at the end.

Reply:
I wanted to be as restrictive as possible to discourage people from adding more until we get a proper solution for the problem. But I dont have a very strong opinion. You think the more generic check is better?

Reply:
If so, we should add comments around `jit_test_functions` to warning the developers in future, and tell them don't add new testing functions. They will see this note when they try to add new items to white list in future.